Listenbrainz/model/Encode:Decode Helpers.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Encode:Decode Helpers.swift
```

Comment:
Remove Encode Decode from the file names

Reply:
done